modules/swagger-codegen/src/main/resources/objc/ApiClient-body.mustache

```diff
@@ -75,7 +75,9 @@ static void (^reachabilityChangeBlock)(int);
         [config.loggingFileHanlder writeData:[message dataUsingEncoding:NSUTF8StringEncoding]];
     }
 
-    NSLog(@"%@", message);
+    if ([[{{classPrefix}}Configuration sharedConfig] debug]){
```

Comment:
@mad102190 would it be better to check the debug flag at the beginning of the method (logResponse) instead and exit the method if debug is false?